components/Op/OpCard.js

```diff
@@ -40,10 +40,10 @@ const OpCard = ({ op }) => {
   const interestIcon = ((interest) => {
     if (!interest) { return '' }
     switch (interest.status) {
-      case 'interested': return <StyledIcon type='like' theme='twoTone' twoToneColor='#6549AA' />
-      case 'invited': return <StyledIcon type='message' theme='twoTone' twoToneColor='#fb0' />
-      case 'committed': return <StyledIcon type='check-circle' theme='twoTone' twoToneColor='#0f0' />
-      case 'declined': return <StyledIcon type='close-circle' theme='twoTone' twoToneColor='#f00' />
+      case 'interested': return <TagState style={{ color: '#222', backgroundColor: '#E1E1E1' }}><StyledIcon type='mail' />You offered to help</TagState>
+      case 'invited': return <TagState style={{ color: 'white', backgroundColor: '#653CAD' }}><StyledIcon type='calendar' />You are invited</TagState>
+      case 'committed': return <TagState style={{ color: 'black', backgroundColor: '#36F482' }}><StyledIcon type='check-circle' />Accepted</TagState>
```

Comment:
At some point we can move these colour definitions to the theme.

Reply:
Can do - just need to move the palette from storybook into css variables